inference-engine/src/low_precision_transformations/src/transformer.cpp

```diff
@@ -316,6 +318,7 @@ TypeRelaxedReplacer::TypeRelaxedReplacer() {
     make_matcher_type_relaxed<opset1::GroupConvolution>(this);
     make_matcher_type_relaxed<opset1::PRelu>(this);
     make_matcher_type_relaxed<opset1::Subtract>(this);
+    make_matcher_type_relaxed<opset1::StridedSlice>(this);
```

Comment:
we don't wrap in TypeRelaxed, do we really need it?

Reply:
My bad. We don't need to update precision in moveDequantizationAfter. Fixed, thanks!